src/environmentd/tests/sql.rs

```diff
@@ -1218,3 +1218,55 @@ fn test_tail_outlive_cluster() {
         0
     );
 }
+
+#[test]
```

Comment:
@mjibson This fails on main.

Reply:
Lol it fails with only 3 loops 3 threads? Ha. Good test.